### ISSUE TITLE
README.md "latest version" is out of date

### ISSUE THREAD

**Issue by danieldides** [terraform-google-modules/terraform-google-sql-db#298]
### TL;DR

The readme states:
> The current version is 3.X.

https://github.com/terraform-google-modules/terraform-google-sql-db/blob/master/README.md?plain=1#L22

However the latest release is currently `v10.0.2`. `3.x` appears to have been released in [late 2019](https://github.com/terraform-google-modules/terraform-google-sql-db/releases/tag/v3.0.0).

Other information such as the [Installation Requirements](https://github.com/terraform-google-modules/terraform-google-sql-db/blob/master/README.md?plain=1#L33) are inconsistent as well. Current it states:

> - [Terraform](https://www.terraform.io/downloads.html) >= 0.13.0
> - [terraform-provider-google](https://github.com/terraform-providers/terraform-provider-google) plugin v2.5.x

Where as the actual latest module requires: 
```
    google = {
      source  = "hashicorp/google"
      version = ">= 4.4.0, < 5.0"
    }
    google-beta = {
      source  = "hashicorp/google-beta"
      version = ">= 4.4.0, < 5.0"
    }
```
https://github.com/terraform-google-modules/terraform-google-sql-db/blob/master/modules/postgresql/versions.tf#L28:L35

### Expected behavior

Readme should be updated or omitted entirely and users directed to refer to the Github Releases and the `versions.tf` files for requirements.

### Terraform Configuration

```hcl
N/A
```

### Terraform Version

```sh
N/A
```

**Comment by bharathkkb** [comment]
Thanks for the report @danieldides. Our automation for updating this is not currently working. Happy to review a PR manually updating this too.
For internal users b/194500618 for context.

**Comment by apeabody** [comment]
https://github.com/terraform-google-modules/terraform-google-sql-db/pull/324 merged - Thanks!
